main.py

```diff
@@ -1,10 +1,49 @@
+import json
+
 import init_django_orm  # noqa: F401
 
 from db.models import Race, Skill, Player, Guild
 
 
 def main() -> None:
-    pass
+    try:
+        with open("players.json", "r") as file:
+            players_data = json.load(file)
+    except FileNotFoundError:
+        print("players.json file not found.")
+        return
+
+    for nickname, player_data in players_data.items():
```

Comment:
Consider adding a check to ensure that `players_data` is a dictionary before iterating over it. This will prevent potential runtime errors if the JSON structure is not as expected.